Source/WebCore/platform/graphics/FloatPoint.h

```diff
@@ -172,6 +172,11 @@ class FloatPoint {
         return { m_y, m_x };
     }
 
+    FloatPoint scaledBy(float scale) const
+    {
+        return FloatPoint(m_x * scale, m_y * scale);
+    }
+
```

Comment:
This method already exists with a slightly different name. Can you reuse that?

https://github.com/WebKit/WebKit/blob/5bff882b42b8d89caf031389f640e95a79b6a01c/Source/WebCore/platform/graphics/FloatPoint.h#L125-L128